pyregion/ds9_region_parser.py

```diff
@@ -152,119 +153,54 @@ def convert_attr(self, l):
                 yield l1, c1
 
     @staticmethod
-    def sky_to_image(l, header, rot_wrt_axis=1):
-        """
+    def sky_to_image(shapelist, header):
```

Comment:
Ah, sorry, I didn't realise this was on an internal method.
Then I agree, no need to point it out in the changelog (unless you've done it already, in this case OK to keep also).

I just remembered that when adding the API docs I also was tempted to clean up argument names for consistency. E.g. here
http://pyregion.readthedocs.io/en/latest/api/pyregion.get_mask.html it's called `region` instead of `shape_list`.
But we can defer this cleanup to separate issues / PRs, this one is mainly about Kapteyn -> Astropy and already very large.